src/Emanote/Pandoc/Renderer/Embed.hs

```diff
@@ -43,14 +43,41 @@ embedWikiLinkResolvingSplice emaAction model nf (ctxSansCustomSplicing -> ctx) _
         B.Div (Url.brokenLinkAttr err) $
           one block
 
-embedSiteRoute :: Monad n => Ema.CLI.Action -> Model -> NoteRenderers n -> HP.RenderCtx n -> WL.WikiLink -> Either MN.Note SF.StaticFile -> Maybe (HI.Splice n)
-embedSiteRoute emaAction model nf RenderCtx {..} wl = \case
+embedInlineWikiLinkResolvingSplice ::
+  Monad n => PandocInlineRenderer n x
+embedInlineWikiLinkResolvingSplice emaAction model nf (ctxSansCustomSplicing -> ctx) _ inl =
+  case inl of
+    B.Link attr@(_id, _class, otherAttrs) is (url, tit) -> do
+      Rel.URTWikiLink (WL.WikiLinkEmbed, wl) <- Rel.parseUnresolvedRelTarget (otherAttrs <> one ("title", tit)) url
+      case Url.resolveWikiLinkMustExist model wl of
+        Left err ->
+          pure $ brokenLinkSpanWrapper err $ B.Link attr (Url.nonEmptyLinkInlines model url Nothing is) (url, tit)
+        Right res -> do
+          embedInlineSiteRoute emaAction model nf ctx wl res
```

Comment:
Okay, as a follow-up to two other comments on `Nothing` further below, I think it would be better to refactor this into two functions. One that takes `note` as argument and returns a block embedding splice (call it `embedResourceRoute`); another that takes `staticFile` as argument and returns an inline embedding splice (call it `embedStaticFileRoute`). 

Then you can use it here as,

```haskell
either (embedResourceNote emaAction model nf ctx wl) (const Nothing) res
```

and in the inline splice,

```haskell
either (const Nothing) (embedStaticFileRoute emaAction model nf ctx wl) res
```